index.bs

```diff
@@ -599,18 +607,18 @@ partial interface MLGraphBuilder {
     therefore its usage is encouraged from the performance standpoint.
     <pre highlight="js">
     const shape = [1,-1,1,1];
-    return builder.add(
+    return builder.setOperand(
+      activation,
```

Comment:
As the sample code, I suppose it could just use one type of activation here, such as `builder.relu`.

Reply:
The implementation of this operation will not know ahead of time what activation is parameterized here. This is why I think we also need to define `setOperand`.

Reply:
I suppose an implementation could know the activation type internally. For example, an implementation may add a internal slot into `MLOperator` that keeps the activation type, e.g. the `MLOperator` instance created by `builder.relu()` has the type `relu`. Later, when the implementation composites the `batchNormalization`, it can check the type of the `MLOperator`.

Reply:
What do you think about moving `setOperand` to the `MLOperator` itself instead of on the builder interface? That way the builder's implementation doesn't need to track the operator type of each operator internally, which could be tedious and ineffective. Instead just have the implementation of the supported operators (e.g. activation functions) support this additional capability themselves without effecting how other operations are implemented. This should address your concern, if I sense it correctly.

Reply:
Moving `setOperand` to `MLOperator` looks better. However my point is either `MLOperator.setOperand/apply` or `MLOperator.type` should be internal to implementation. If there are no use cases for user code, I would suggest exclude it in this PR.

Reply:
An implementation that doesn't want to handle composite operations like `batchNorm` natively should be able to implement it as if it's a graph of smaller operations, like what being described in the Note section. Remember that that is one of our design principles -- that every big op defined must be publicly implementable by compositing smaller public constructs? 

Although we do expect operations like `batchNorm` to be implemented natively, some implementation on some platform may not have a mean to do that, and would only settle on a default implementation as a graph of public operations.

Reply:
> An implementation that doesn't want to handle composite operations like `batchNorm` natively should be able to implement it as if it's a graph of smaller operations,  like what being described in the Note section.

How about the Note section says "When the input tensor is 4-D of the "nchw" layout and ___the activation is a relu operator___, the behavior of this operation can be generically emulated from the usage of other operations as follow."

```js
const shape = [1,-1,1,1];
return builder.relu(
  builder.add(
    builder.mul(
      builder.reshape(options.scale, shape),
      builder.div(
        builder.sub(input, builder.reshape(mean, shape)),
        builder.pow(
          builder.add(builder.reshape(variance, shape), builder.constant(options.epsilon)),
          builder.constant(0.5))
        )),
    builder.reshape(options.bias, shape)));
```

> some implementation on some platform may not have a mean to do that,

That's true. For example, the webnn-polyfill implementation uses TF.js's [`tf.batchNorm`](https://js.tensorflow.org/api/latest/#batchNorm) that doesn't support fused activation. 

> and would only settle on a default implementation as a graph of public operations

This might not be true. For example, webnn-polyfill implements WebNN spec defined ops as one or multiple TF.js kernels. For fused `batchNormalziation`, it could call the corresponding Tf.js kernel of an `MLOperator`, e.g. `tf.relu`, right after `tf.batchNorm`. That doesn't require the `setOperand` as a public API.

Reply:
@wchao1115 , I put my idea into https://github.com/webmachinelearning/webnn/pull/193, please take a look. Thanks.

Reply:
Thanks for the feedback @huningxin . Can you please take a look at the new commit? I adapted your suggestion in this update.